crossfire/clients/occurrences.py

```diff
@@ -14,12 +16,27 @@
 except ImportError:
     pass
 
-from crossfire.errors import CrossfireError, RetryAfterError
+from crossfire.errors import (
+    CrossfireError,
+    InvalidDateFormatError,
+    InvalidDateIntervalError,
```

Comment:
Você acha que _inválido_ e _erro_ juntos é redundância? Estava pensando aqui…

```suggestion
    DateFormatError,
    DateIntervalError,
```

Reply:
Cara, concordo. Acho redundancia, sim. Isso estava me incomodando... e não sei o motivo de não ter me manifestado antes.